test/blackbox-tests/test-cases/inline_tests/compile-flags.t/dune-file-with-valid-flag/dune

```diff
@@ -0,0 +1,12 @@
+(library
```

Comment:
I think you could probably just have one dune file with a single dune file, and have the two different libs use the same backend.

Reply:
sounds reasonable.